Python/getargs.c

```diff
@@ -1973,16 +1973,11 @@ new_kwtuple(const char * const *keywords, int total, int pos)
     return kwtuple;
 }
 
-static int
-parser_init(struct _PyArg_Parser *parser)
+static inline int
+_parser_init(struct _PyArg_Parser *parser)
```

Comment:
This function seems awfully big to be inlined.  It may make more sense to inline the top-level `parser_init()` instead.